GetStream/stream-chat-android
# [CAS-1403] Add handling max attachments count

https://stream-io.atlassian.net/browse/CAS-1403

### 🎯 Goal

Add handling max attachments count

### 🛠 Implementation details

- A single message cannot contain more than 10 attachments. Added a warning when the user picks more than 10 attachments in `MessageInputView`. From now on, it will be also impossible to send messages with > 10 attachments.
- Maximum attachments count is configurable
- Default error message is configurable using `SelectedAttachmentsCountListener `

### 🎨 UI Changes

<img width="210" alt="Screenshot 2021-10-22 at 10 58 10" src="https://user-images.githubusercontent.com/17440581/138429721-198314ee-0b78-4ad5-95e6-0c53cbe77621.png">

### 🧪 Testing

Play with `MessageInputView`:
- Try to add more than 10 attachments - you should see an error
- Try to send a message with more than 10 attachments
- Play with `streamUiMaxAttachmentsCount ` attr and check if it works correctly
- Try to customize the error message using `SelectedAttachmentsCountListener`

### ☑️ Checklist
